_docs/_latest/develop/test/integration-testing/run-test-suite.md

```diff
@@ -0,0 +1,148 @@
+* billing.accounts.setIamPolicy on a billing account.
+- `Billing Account Administrator` role should be granted to the admin account to 
+be able to create the Service Account using the helper script. You can revoke 
+this role after the service account is created.
+- To run the [helper script](https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/helpers/setup-sa.sh), 
+clone the [Terraform Google Project factory repository](https://github.com/terraform-google-modules/terraform-google-project-factory)
+and run the following command:
+
+`./helpers/setup-sa.sh <ORGANIZATION_ID> <PROJECT_NAME> <BILLING_ACCOUNT>`
+
+### **Creating Service Account and granting roles manually**
+
+Alternatively, you can grant the following roles and enable the APIs manually.
+
+On the organization:
+
+* roles/resourcemanager.organizationAdmin
+* roles/iam.securityReviewer
+* roles/resourcemanager.folderViewer
+* roles/resourcemanager.organizationViewer
+* roles/resourcemanager.projectCreator
+* roles/billing.user
+
+On the project:
+
+* roles/owner
+* roles/compute.instanceAdmin
+* roles/compute.networkViewer
+* roles/compute.securityAdmin
+* roles/iam.serviceAccountAdmin
+* roles/serviceusage.serviceUsageAdmin
+* roles/iam.serviceAccountUser
+* roles/storage.admin
+* roles/cloudsql.admin
+
+Enable the following APIs on the Forseti project:
+
+* cloudresourcemanager.googleapis.com
+* compute.googleapis.com
+* serviceusage.googleapis.com
+
+On the host project when using shared VPC
+
+* roles/compute.securityAdmin
+* roles/compute.networkAdmin
+* roles/browser
+* roles/resourcemanager.projectIamAdmin
+
+On the organization when using shared VPC
+
+* roles/compute.securityAdmin
+* roles/compute.networkAdmin
+* roles/compute.xpnAdmin
+
+### **Setting up environment variables**
+
+Set the following environment variables from the bash shell:
+
+```
+export SERVICE_ACCOUNT_JSON=<JSON_KEY_OF_THE_NEWLY_CREATED_SERVICE_ACCOUNT> \
+
+export TF_VAR_billing_account=<YOUR_BILLING_ACCOUNT> \
+
+export TF_VAR_domain=<YOUR_DOMAIN> \
+
+export TF_VAR_forseti_email_recipient=<EMAIL_RECIPIENT> \
+
+export TF_VAR_forseti_email_sender=<EMAIL_SENDER> \ 
+
+export TF_VAR_forseti_version=<BRANCH_NAME> \ 
+
+export TF_VAR_gsuite_admin_email=<GSUITE_EMAIL_ADDRESS> \
+
+export TF_VAR_inventory_performance_cai_dump_paths=<PATH_TO_CAI_DUMP> \
+
+export TF_VAR_kms_key=<KMS_KEY> \
+
+export TF_VAR_kms_keyring=<KMS_KEYRING> \ 
+
+export TF_VAR_org_id=<YOUR_ORGANIZATION_ID> \
+
+export TF_VAR_project_id=<YOUR_PROJECT_ID> \
+
+export TF_VAR_sendgrid_api_key=<SENDGRID_API_KEY>
+
+```
+
+## **Running the test suite**
+
+Run the following command after setting up environment variables:
+
+```
+docker container run -it -e KITCHEN_TEST_BASE_PATH="integration_tests/tests" -e 
+SERVICE_ACCOUNT_JSON -e TF_VAR_project_id -e TF_VAR_org_id -e 
+TF_VAR_billing_account -e TF_VAR_domain 
+-v $(pwd):/workspace gcr.io/cloud-foundation-cicd/cft/developer-tools:0.4.1
+/bin/bash
```

Comment:
The additional env vars mentioned above also need to be passed in.

Reply:
Done.